rpcs3/Emu/RSX/VK/VKFragmentProgram.cpp

```diff
@@ -228,43 +228,17 @@ void VKFragmentDecompilerThread::insertMainStart(std::stringstream & OS)
 		{
 			if (m_prog.front_back_color_enabled)
 			{
-				if (PI.name == "spec_color")
+				if (PI.name == "spec_color" && m_prog.back_color_specular_output && m_prog.front_color_specular_output)
```

Comment:
I see. Maybe others should check gl_FrontFacing too? Like `gl_FrontFacing ? back_diff_color : something_else`, maybe constant or something.